Assets/Plugins/LeapMotion/Core/Shaders/InteractionVolumeWireframe.shader

```diff
@@ -0,0 +1,100 @@
+﻿Shader "LeapMotion/Geometry/Wireframe"
```

Comment:
The `.meta` files for all of the added assets were ignored in the commits.  This is bad; they should be committed, as they are how Unity stores asset GUIDs and defines inter-file connections.   It is particularly egregious for this file because the Interaction Zone Material contains a broken reference to this shader, which prevents the interaction volume from drawing.   Please commit the generated `.meta` files with the assets!